packages/flutter_tools/test/commands.shard/permeable/analyze_once_test.dart

```diff
@@ -138,8 +157,9 @@ void main() {
         exitMessageContains: '3 issues found.',
         toolExit: true,
       );
+      optionsFile.deleteSync();
```

Comment:
This should probably go under a `finally {}`.

Reply:
Done